index.html

```diff
@@ -88,10 +88,6 @@ <h2>Jump into using Caliper with the other projects</h2>
         <strong>Hyperledger Besu</strong>
         <img src="https://wiki.hyperledger.org/download/thumbnails/16326961/Hyperledger_Besu_color.png?version=1&modificationDate=1567198581000&api=v2" alt="Burrow">
       </a>
-      <a class="project" href="{{ site.baseurl }}/{{ site.caliper.current_release }}/burrow-config">
-        <strong>Hyperledger Burrow</strong>
-        <img src="https://www.hyperledger.org/wp-content/uploads/2018/06/Hyperledger_Composer_Logo_White.png" alt="Burrow">
-      </a>
       <a class="project" href="{{ site.baseurl }}/{{ site.caliper.current_release }}/ethereum-config">
         <strong>Ethereum</strong>
```

Comment:
The text is next to the image, looks a bit weird.

Reply:
hmmm, that one is a sizing issue and likely to be a style thing with flex rather than a text placement issue

Reply:
addition of `display: flex` and `flex-direction: column` 👍